src/System.Net.NetworkInformation/tests/FunctionalTests/PhysicalAddressTest.cs

```diff
@@ -103,7 +103,8 @@ public void Parse_Valid_Success(string address, byte[] expectedBytes)
         [InlineData("AE88.D6EC.A720")]
         public void Parse_Invalid_ThrowsFormatException(string address)
         {
-            Assert.Throws<FormatException>(() => PhysicalAddress.Parse(address));
+            FormatException ex = Assert.Throws<FormatException>(() => PhysicalAddress.Parse(address));
+            Assert.Contains(address, ex.Message);
```

Comment:
Why causes this the `Linux arm64 Release Build` to fail? 🤔😕

Reply:
It's unrelated, some kind of dns issue on an Ubuntu vm:
https://mc.dot.net/#/user/Smartis2812/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/14f0e657372567638e375050ecb5b626f50fa123/workItem/System.Net.NameResolution.Pal.Tests/analysis/xunit/System.Net.NameResolution.PalTests.NameResolutionPalTests~2FTryGetAddrInfo_HostName